src/libs/actions/Report.ts

```diff
@@ -2790,8 +2801,27 @@ function leaveGroupChat(reportID: string) {
         });
     }
 
+    const successData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.REPORT}${reportID}`,
+            value: Object.keys(report).reduce<Record<string, null>>((acc, key) => {
```

Comment:
This block led to the following issue:
- https://github.com/Expensify/App/issues/52924

which was fixed by setting value to `null` - more details in the selected [proposal](https://github.com/Expensify/App/issues/52924#issuecomment-2492795635).